### ISSUE TITLE
Can't share on Facebook, please link your account!

### ISSUE THREAD

**Issue by LePoissonBleu** [DeathMiner/RewardsGG-Farm#21]
Accound is linked. 

Here a copy paste of a log : 

=> REWARDS.GG FARM v2.3
   Using RewardsGG site v1.7.1b1 [tested on v1.7.0b1]
   <3 from Death_Miner
[6**] Can't share on Facebook, please link your account!
[6:**] Earned 20 tickets!
[6:**] Can't share on Facebook, please link your account!
[6:**] Earned 20 tickets!
[6:**] Can't share on Facebook, please link your account!
[6:**] Can't share on Facebook, please link your account!
[6:**] Earned 20 tickets!
[6:**] Can't share on Facebook, please link your account!
[6**] Earned 20 tickets!
[6:**] Can't share on Facebook, please link your account!
[6:**] Earned 20 tickets!
[7:**] Earned 20 tickets!
[7:**] Can't share on Facebook, please link your account!

**Comment by demaisj** [comment]
Hi,

Can you send me a picture of the JavaScript Console when the FB sharing is happening please ?
I can't even test an update I just got my account locked lol

**Comment by LePoissonBleu** [comment]
yeah, Im about to go to sleep. I will send you a picture tomorow.

On a side note, you should really take a look at winaskin.com They work in a similar way, but give more token on ads and the newly updated site give cheap skins overtime.

**Comment by SantuZ** [comment]
Deathminer,
Here is the screenshot of the problem that LePoissonBleu stated:
![capture](https://cloud.githubusercontent.com/assets/20721369/17243306/a6099760-5584-11e6-809d-60d12d36321e.PNG)

**Comment by LePoissonBleu** [comment]
@DeathMiner 

The error you are looking for is the following:

VM3746 extensions::messaging:342 Uncaught TypeError: Cannot read property 'name' of undefined

**Comment by demaisj** [comment]
Don't you have a complete screenshot of the JavaScript console pls? I need to see everything happening in the console at the moment of the problem.

**Comment by SantuZ** [comment]
@DeathMiner  Here you go:
![capture](https://cloud.githubusercontent.com/assets/20721369/17259878/2a5cfe98-55d6-11e6-9ab3-6d8e33db6bb5.PNG)

**Comment by demaisj** [comment]
This error doesn't come from the farm itself. The farm here is VM16507 and the error comes from VM16478_extensions:messaging.
Plus you didn't do a screenshot where the Facebook sharing was happening. (It should log Can't share account not linked + other useful infos)

**Comment by LePoissonBleu** [comment]
Hum, it is strange.

To be clear, it does not do the sharing on facebook. Im unsure what else I can provide. I even tried leaving a facebook tab open. Nothing seem to work.

**Comment by LePoissonBleu** [comment]
[REWARDS.GG FARM] [6:27:34] Can't share on Facebook, please link your account!
VM10485:314 FB SHARE: You must link your account (My account - Link my account)
VM10485:368 TW FOLLOW: Vous devez suivre 'RewardsGG'
VM10485:349 G2A CLICK: Come after 3 days 7 hours
VM10485:387 STEAM USERNAME: Votre pseudo Steam ne contient pas rewards.gg (<a href="http://steamcommunity.com/MYPROFILE(EDITED OUT)"  target="_blank">Changer votre nom de profil Steam ?</a>
VM10485:177 [REWARDS.GG FARM] 225/780
VM10485:333 TW SHARE: You have already retweeted this tweet.

**Comment by kaiserdj** [comment]
I have the same problem

**Comment by LePoissonBleu** [comment]
I understand you got your account banned and unaware if you have a new account and you may be unable to test/help atm. Just figured I would share something I considered.

If the facebook share feature work on your computer and no ours. Could it be possible that is has something to do with windows installation and language impact it has on websites ? My windows is in french and it make facebook french.

**Comment by demaisj** [comment]
Hi, I already talked about this in issue #23
